app/views/admin/event_types/index.html.haml

```diff
@@ -39,4 +39,3 @@
 .row
   .col-md-12.text-right
     = link_to 'Add Event Type', new_admin_conference_program_event_type_path(@conference.short_title), class: 'btn btn-primary'
-
```

Comment:
When I save a file in atom if there is a empty line at the end it removes it. I'll exclude that from the PR.